pkg/config/v2/config_loader.go

```diff
@@ -444,14 +447,18 @@ func getType(typeDef typeDefinition) (Type, error) {
 		return EntityType{
 			EntitiesType: typeDef.Entities.EntitiesType,
 		}, nil
+	case typeDef.isAutomation():
+		return AutomationType{
+			Resource: typeDef.Automation.Resource,
+		}, nil
 
 	default:
-		return nil, fmt.Errorf("invalid typeDefinition - is neither Setting, Classic nor Entity")
+		return nil, fmt.Errorf("invalid typeDefinition - it's neither Setting, Classic, Automation, nor Entity")
```

Comment:
Just report with "invalid typeDefinition". The rest (list of known) is horible to maintain and have tendency to produce missleading information